decidim-core/db/migrate/20161108093802_create_decidim_pages.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDecidimPages < ActiveRecord::Migration[5.0]
+  def change
+    create_table :decidim_pages do |t|
+      t.hstore :title, null: false
+      t.string :slug, null: false
```

Comment:
Missing index to prevent duplication of pages with the same slug in the same organization